agent/task.go

```diff
@@ -1,12 +1,12 @@
 package agent
 
 import (
-	"reflect"
 	"time"
 
 	"github.com/docker/swarmkit/agent/exec"
 	"github.com/docker/swarmkit/api"
 	"github.com/docker/swarmkit/log"
+	"github.com/docker/swarmkit/manager/dispatcher"
```

Comment:
Thanks Aaron. This functions should be shared by agent and dispatcher. I don't find a good location for such a shared package. Any suggestion?